.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -35,4 +33,4 @@ body:
       options:
         - label: If I have identified that the work is specific to a repository, I have removed "repository:addons-server" or "repository:addons-frontend"
           required: true
```

Comment:
If you're removing the `"repository:addons-server", "repository:addons-frontend"` labels, this should be removed as well right ?